src/components/Items/Items.js

```diff
@@ -92,18 +107,66 @@ export class Items extends Component {
       storeBlock = (
         <h3>
           {store.name}
+
           {address}
         </h3>
       );
     }
+    let header;
+    let editForm = (
+      <form onSubmit={e => this.newName(e)}>
+        <input
+          type="text"
+          ref={editListName => {
+            this.editListName = editListName;
+          }}
+        />
+        <button>
```

Comment:
prevent submission if field is empty - at the moment, it's allowing no-name update